docs_user/modules/proc_migrating-tls-everywhere.adoc

```diff
@@ -48,7 +43,10 @@ endif::[]
 $IPA_SSH certutil -L -d /etc/pki/pki-tomcat/alias
 ----
 +
-The `-L` option lists all certificates, and `-d` specifies where they are stored. This will produce some output like this:
+* The `-L` option lists all certificates.
+* The `-d` option specifies where the certificates are stored.
++
```

Comment:
Are the -L option and -d options common knowledge for any customers following this procedure?

Reply:
not sure - probably not.  Most folks are probably unfamiliar with certutil.